Subbus/Classes/Subbus.PersistentEvent.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  Subbus.PersistentEvent.swift
+//  Subbus
+//
+//  Created by Jangle's MacBook Pro on 3/15/21.
+//
+
+/// A special event type that Subbus will persist beyond `Post`ing the event.
+public class PersistentEvent {
```

Comment:
I do not understand how this is supposed to be used. Do we subclass? 

How do I use this in conjunction with scoping?

Reply:
Check out the README!

As far as conjuction with scoping: I'd actually like to refactor scoping, as it's not currently compatible with Subbus.  It's neat that we made it "separate" from Subbus, but that also means it can't be used in conjunction with anything else.  It is already its own event.

Reply:
Update: I made [an issue](https://github.com/myriadmobile/subbus/issues/6) for this